windows/Makefile

```diff
@@ -55,11 +65,41 @@ extra_drivers:
 .INTERMEDIATE: extra_drivers http/Autounattend.xml
 
 windows: check-deps clean extra_drivers http/Autounattend.xml OVMF_VARS.fd
+ifeq ($(strip $(VERSION)),11)
+	mkdir -p /tmp/swtpm; \
+	swtpm socket --daemon --tpm2 --tpmstate dir=/tmp/swtpm --ctrl type=unixio,path=/tmp/swtpm/swtpm-sock --log level=10
+endif
+ifneq ($(strip $(VHDX)),)
+	qemu-img convert -p -O raw ${VHDX} windows_builder.img; \
```

Comment:
this could be a separate shell script, it would be clearer to understand

Reply:
Noted, will add to TODO